tests/Tests/100__deploy/100__installation/103__post_install_monitoring_port_forwarding.robot

```diff
@@ -0,0 +1,52 @@
+*** Settings ***
+Library          OperatingSystem
+Library          Process
+Resource         ../../../Resources/Page/OCPDashboard/Monitoring/Grafana.robot
+Resource         ../../../Resources/ODS.robot
+Resource         ../../../Resources/Common.robot
+Suite Setup      Set Library Search Order  SeleniumLibrary
+Suite Teardown   Close Browser
+
+
+*** Variables ***
+${PROC}                         none
+${JUPYTERHB_SLI}                //*[contains(@aria-label,"Jupyterhub SLIs")]
+
+
+*** Test Cases ***
+Verify That MT-SRE Can Connect To Grafana Using Port Forwarding
+    [Documentation]    Verifies that Grafana is accessible by MT-SRE when using oc port-forwarding
+    [Tags]   Tier2
+    ...      ODS-754
+
+    # Enable Port Forwarding
```

Comment:
Commenting code is discouraged. The keywords should be meaningful by themselves

Reply:
Updated